dynamiq/utils/utils.py

```diff
@@ -13,6 +13,8 @@
 
 TRUNCATE_LIMIT = 20
 
+INCLUDE_NONE_KEYS = frozenset({"input", "output"})
```

Comment:
- Please use tuple
- let's use more convenient name for this. This is unclear of purpose, where used, etc. Does it make sense to move to Runnable as some ClassVar, with potential extending/overriding for each node/flow/workflow?